src/client/components/app/Transactions/components/GenerateClaimRevenueInputs.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
can u remove Generate from the file/component name plz

Reply:
modified to `TxClaimRevenueInputs` to match naming convention of other files in the directory